serverless/virus-scanner/src/index.ts

```diff
@@ -0,0 +1,109 @@
+import { APIGatewayProxyResult } from 'aws-lambda'
+import crypto from 'crypto'
+import { StatusCodes } from 'http-status-codes'
+
+import { scanFileStream } from './clamscan.service'
+import { config } from './config'
+import { getLambdaLogger } from './logger'
+import { S3Service } from './s3.service'
+import { isBodyWithKey } from './types'
+
+export const handler = async (
+  event: unknown,
+): Promise<APIGatewayProxyResult> => {
+  const logger = getLambdaLogger('virus-scanner')
+
+  logger.info('handler triggered')
+
+  if (!isBodyWithKey(event)) {
+    logger.warn({
+      message: 'Missing key in body',
+      event,
+    })
+    return {
+      statusCode: StatusCodes.BAD_REQUEST,
+      body: JSON.stringify({
+        message: 'Missing key in body',
+      }),
+    }
+  }
+
+  const quarantineFileKey = event.key
```

Comment:
Should we add a validation that `event.key` is a UUID and nothing else?

Reply:
done in [152d666](https://github.com/opengovsg/FormSG/pull/6642/commits/152d66686ce75ae8b47c22076af9390903de7a72) / [a0c4a17](https://github.com/opengovsg/FormSG/pull/6642/commits/a0c4a173c5fa15df9311a5094d64a097ecb605ba)